plutok-flat/src/main/kotlin/io/newm/plutok/uplc/ast/PlutusData.kt

```diff
@@ -0,0 +1,26 @@
+package io.newm.plutok.uplc.ast
+
+import java.math.BigInteger
+
+sealed class PlutusData
```

Comment:
Same here. This can be an `sealed interface`.

Reply:
fixed all of these